components/contribution-flow/index.js

```diff
@@ -140,8 +144,9 @@ class ContributionFlow extends React.Component {
       stepDetails: {
         quantity: 1,
         interval: props.fixedInterval || getDefaultInterval(props.tier),
-        amount: props.fixedAmount || getDefaultTierAmount(props.tier),
+        amount: props.verb === 'crypto' ? 5 : props.fixedAmount || getDefaultTierAmount(props.tier),
```

Comment:
What is the magic value `5`?

Reply:
Yes, as discussed I've defaulted this to 0 and fixed the behavior such that when the use first visits this section the input box is auto focused and when the focus out of it the validation takes place. 👍🏽 

![validation-on-input](https://user-images.githubusercontent.com/12435965/128557804-133f7544-3914-4225-a8b9-ea38486bb698.gif)

Reply:
I don't see zero has a good default value either. Could it perhaps be empty instead?

Reply:
Yes, it's possible and I've changed this to have an empty default value. 👍🏽